Libraries/Drei/core-commands.lisp

```diff
@@ -437,9 +437,12 @@ will ask whether to sort in ascending or descending order."
 ;; Copies an element from a kill-ring to a buffer at the given offset
 (define-command (com-yank :name t :command-table editing-table) ()
   "Insert the objects most recently added to the kill ring at point."
-  (handler-case (insert-sequence (point) (kill-ring-yank *kill-ring*))
-    (empty-kill-ring ()
-      (display-message "Kill ring is empty"))))
+  (alexandria:if-let
```

Comment:
[Under some circumstances, yanking will insert top entry in the kill ring rather than requesting the clipboard content from the owner. One way to reproduce this issue is to open an application with an editor and copy some text. Then close the application (without restarting the Lisp runtime), copy some text from a different client and then restart the CLIM application. Pasting will not paste the clipboard and instead paste from the kill ring.

At this point, McCLIM will ignore any copy from any application, and will only start working again after explicitly copying some content within McCLIM.

Reply:
irrelevant (this issue is present in master branch), see tickets (reported it)